system.tex

```diff
@@ -17,31 +19,39 @@ \subsection{Data verification and quality assurance }
 They require access to tools installed at the summit as well as in the data facilities.
 In particular they will need access to images as they are processed to generate alerts.
 
-These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by (need doc).\action{RD}{Mar 2022}{Handle for doc covering users accounts at \gls{SLAC}}.
+These team members will have to have \gls{SLAC} accounts to perform this work - \gls{SLAC} accounts are governed by SLAC procedures \cite{SLACOB} and \cite{SLACNH}.
 In addition they will need summit accounts as governed by \citeds{ittn-045} and \citeds{ittn-010}.
 
 
 
 \subsection{Data processing}
```

Comment:
I am confused why we skip prompt processing and go straight to DRP. PP is the more relevant for security.

Reply:
added a section for that .. good catch